ctypes/0x5fd43243e69170a86159ec68e8913a34a1e676e237adde0f86ed6e4a5b08649b/ctype.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://kilt-protocol.org/draft-01/ctype#",
+    "properties": {
+        "comment": {
+            "type": "string"
+        }
+    },
+    "title": "Comment",
```

Comment:
Can we maybe give it a more indicative name? I know this has already been written on chain, but just "Comment" does not say much.

Reply:
ideally we would review CTypes before they hit the chain.

Reply:
How would we do that?

Reply:
Yeah I didn't phrase that good. I mean that we review our own ctypes like we review our code.
Others can submit what ever.

Reply:
I don't see how that isn't a good name. They perfectly describe what they intend.

Reply:
So you are assuming that all platforms in the world, when they want to deal with comments, they would be dealing with a single blob of text? I think this is much of an opinionated structure for this specific use case, that is why I was suggesting that the name `Comment` might be too generic. If every platform creates its own `Comment` CType, then the title itself becomes meaningless.

Reply:
I love the name. Concise, to the point. Whoever came up with this name is a CType poet.

Reply:
But maybe it's even too descriptive. `String` could have been a better title

Reply:
Ideally, they wouldn't create a new CType. At the moment, the perfect comment CType would be something other than something that would be achievable with our current restraints on the meta schema. 

If you wanted more types, we would have been able to manage it by including `oneOf` in the Meta Schema, allowing us to select which type.

Similar to the example below.

```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "type": "object",
  "required": ["results"],
  "additionalProperties": false,
  "properties": {
    "results": {
      "type": "object",
      "properties": {
        "result": {
          "oneOf": [
            {
              "type": "string",
              "enum": ["1", "2", "3", "4"]
            },
            {
              "type": "integer",
              "minimum": 1,
              "maximum": 4
            }
          ]
        }
      }
    }
  }
}
```

Unfortunately, we don't have this feature to a generic data blob for comment should be a good beginning point since we know that CTypes are versionable.

Reply:
My point was not so much about the structure, as it was about the naming.